### PR TITLE
TOK-410: all time share

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

- All time share

# Link

- [TOK-409](https://rsklabs.atlassian.net/browse/TOK-409)

### CODE REVIEW

**Review by antomor — APPROVED** [review]
LGTM

**Comment by jurajpiar** [comment]
Screenshot 2024-11-12 at 11 39 26
Do you plan on centring the text it in this PR?

File: src/app/collective-rewards/rewards/AllTimeShare.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  const error = rewardsError ?? gaugeRewardsError ?? claimableRewardsError
+
+  useHandleErrors({ error, title: 'Error loading all time share' })
+
+  let isLoading: boolean = logsLoading || gaugeLogsLoading || claimableRewardsLoading
+
+  if (!rewardsPerToken[rif.address] || !gaugeRewardsPerToken[rif.address]) {
+    return (
+      <MetricsCardWithSpinner isLoading={isLoading} borderless>
+        <MetricsCardTitle title="All time share" data-testid="AllTimeShare" />
```
Comment (antomor) [inline src/app/collective-rewards/rewards/AllTimeShare.tsx:63]: @franciscotobar Should we encolse title and content into a div as we do below?

**Review by antomor — APPROVED** [review]
LGTM

File: src/app/collective-rewards/rewards/AllTimeShare.tsx (hunk 1 above)
Comment (antomor) [inline src/app/collective-rewards/rewards/AllTimeShare.tsx:63]: @franciscotobar Should we enclose it in a div as we do below?

**Pull request merged**